src/Utils/CameraUtils.js

```diff
@@ -337,7 +337,7 @@ class CameraRig extends THREE.Object3D {
 }
 
 export function getRig(camera) {
-    rigs[camera.uuid] = rigs[camera.uuid] || new CameraRig(camera);
+    rigs[camera.uuid] = rigs[camera.uuid] || new CameraRig();
```

Comment:
Already changed by #2107, rebase from master.
```suggestion
    rigs[camera.uuid] = rigs[camera.uuid] || new CameraRig(camera);
```